src/components/ReceiverForm/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
+import { Container, Form, Button, Col, Row } from "react-bootstrap";
+import db from "../../firebaseConfig";
+import "./index.css";
+
```

Comment:
The receiver and donator forms look similar to me, instead you can do only one form, and change what’s being posted to firebase according to the props coming to the component, you don’t need to repeat yourself